md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/CrawlerAPI.java

```diff
@@ -46,16 +45,15 @@
 @Scope("session")
 public class CrawlerAPI extends MetadataAPIBase {
     private static final Logger LOGGER = Logger.getLogger(CrawlerAPI.class);
+    private static final String CRAWLER = "crawler";
+
     @Autowired
     private ProcessDAO processDAO;
-    @Autowired
-    private PropertiesDAO propertiesDAO;
 
     @RequestMapping(value = {"/", ""}, method = RequestMethod.POST)
 
-    public
     @ResponseBody
-    RestWrapper createCrawlerProcess(@ModelAttribute("crawlerInfo")
+    public RestWrapper createCrawlerProcess(@ModelAttribute("crawlerInfo")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "createCrawlerProcess" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)